Supporting files/Info.plist

```diff
@@ -41,5 +41,10 @@
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
+	<key>NSAppTransportSecurity</key>
```

Comment:
Just a thing to remember: in the real app it wouldn't be possible to ship an app tp the App Store with this thing :) But for us now it's fine 👍